### PR TITLE
Import gatsby source airtable

### LINKED ISSUE #368: add[gatsby-source-airtable]
### Plugin Repository

https://github.com/jbolda/gatsby-source-airtable

### Plugin Package

https://www.npmjs.com/package/gatsby-source-airtable

### Plugin Description

Sources data from Airtable. 

### Further Details

https://github.com/gatsby-uc/plugins/discussions/367

Chatted with @jbolda and he's busy on other projects and has previously tried to get Gatsby and Airtable to take over this important plugin. They've both declined and Jacob is happy to have it find a home here in the GUC. 

#### Contributor Tasks: 
- [x] [transfer the repo](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#transferring-a-repository-owned-by-your-user-account) to the `gatsby-uc` 
- [x] [Add `moonmeister` as a maintainer](https://docs.npmjs.com/transferring-a-package-from-a-user-account-to-another-user-account) on your npm package. 

#### Admin 
- [x] Move code into `gatsby-uc/plugins` repo
- [x] Update Readme in old repo to redirect folks to new location using [template](https://github.com/gatsby-uc/plugins/blob/main/templates/old_repo_readme.
- [x] Move/Comment on still applicable PRs/
- [ ] Archive old 
- [ ] Update Plugin with [best practices](https://github.com/gatsby-uc/plugins/blob/main/CONTRIBUTING.md#plugin-best-
- [ ] Publish new version

### PR DESCRIPTION
**Pull request by jbolda**
## Description

Moving the `gatsby-source-airtable` plugin into the collectives umbrella as noted in #368.

## Related Issues

Addresses a bunch of the tasks in #368.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -4670,6 +4670,73 @@ __metadata:
   languageName: node
   linkType: hard
```
Comment (moonmeister) [inline yarn.lock:4671]: This file can be eliminated since it's now in a monorepo. Make sure to run `yarn` to update the repo lock file.

**Review by moonmeister — COMMENTED** [review]
I'm on my phone and this was a quick scroll, but thought I'd provide quick comments as I'm not sure when I'll have time to review more fully. Looks like step one was just move the code, not necessarily conform it to the new repo, so you may be aware of what I raised

File: packages/gatsby-source-airtable/LICENSE [hunk 2]
```diff
@@ -0,0 +1,20 @@
+MIT License
```
[the remark this answers is not in the document]
Comment (jbolda) [reply packages/gatsby-source-airtable/LICENSE:1]: I left it in for the copyright.

File: yarn.lock (hunk 1 above)
Comment (jbolda) [reply yarn.lock:4671]: Ah, this is the repo lock file.

File: integration-tests/source-airtable/recipes-with-multi-type/.npmrc [hunk 3]
```diff
@@ -0,0 +1 @@
+legacy-peer-deps = 'true'
```
[the remark this answers is not in the document]
Comment (jbolda) [reply integration-tests/source-airtable/recipes-with-multi-type/.npmrc:1]: Yea, I can remove these and the `package-lock.json`. I was trying not to get to deep into updating the examples as I imagine that is a rabbit hole.

**Comment by jbolda** [comment]
I ignored the bulk of the eslint errors as many of those would either substantially change the code or possibly change the API response which would be a major bump. I figure it makes sense that this PR focus strictly on the import and further updates to the examples or code could be done in separate, more focused PRs.

File: packages/gatsby-source-airtable/README.md [hunk 4]
```diff
@@ -0,0 +1,379 @@
...
+
+[![gatsby-source-airtable](./header.png)](.)
+
+# gatsby-source-airtable
+
+Gatsby source plugin for sourcing data into your Gatsby application from your
+Airtable base tables
+
+[![npm](https://badgen.net/npm/v/gatsby-source-airtable)](https://www.npmjs.com/package/gatsby-source-airtable)
+[![travis](https://badgen.net/travis/jbolda/gatsby-source-airtable)](https://travis-ci.com/jbolda/gatsby-source-airtable)
```
Comment (moonmeister) [inline packages/gatsby-source-airtable/README.md:12]: Should we remove this badge?

File: packages/gatsby-source-airtable/package.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  ],
+  "keywords": [
+    "airtable",
+    "gatsby",
+    "gatsby-plugin",
+    "gatsby-source",
+    "gatsby-source-airtable"
+  ],
+  "scripts": {
+    "test": "echo \"Error: no test specified\""
```
Comment (moonmeister) [inline packages/gatsby-source-airtable/package.json:24]: This should execute a test if we want them to run on our current CI config.

**Review by moonmeister — CHANGES_REQUESTED** [review]

File: packages/gatsby-source-airtable/package.json (hunk 5 above)
Comment (jbolda) [reply packages/gatsby-source-airtable/package.json:24]: The "tests" we had were running it within the examples. There are no unit tests.

Comment (moonmeister) [reply packages/gatsby-source-airtable/package.json:24]: I totally understand the separation. Probably how we should do the rest of the integration tests. I can work on figuring this out but we just need to change CI to run integration tests as a separate CI job. Part of me thinks this should be a separate PR, but I also want it for this so we actually get the tests passing for this PR. Thoughts?

Comment (moonmeister) [reply packages/gatsby-source-airtable/package.json:24]: So Just looking through the integration tests and I'm not seeing any "test" scripts there either. How are you expecting to execute tests?

Comment (jbolda) [reply packages/gatsby-source-airtable/package.json:24]: I kind of feel like it is a slippery slope. If you start running the builds, you might run into another thing that should be fixed and a rabbit hole of things to update. I didn't include a change file so this should publish a new release. It feels like there is the opportunity there to do some updating separate from this PR.

Ah, so the examples don't have a test script either. They are pretty horribly styled examples that are structured to show one or many of the features of the source plugin. The idea is that the `gatsby build` would straight up fail if there was an issue. This was kind of the best path that I found to maintain it while working with v0, v1 and v2.

Comment (moonmeister) [reply packages/gatsby-source-airtable/package.json:24]: Sounds good. I think I'm comfortable with this as is. Maybe we create an issue to resolve testing in the future.

**Review by moonmeister — APPROVED** [review]

**Pull request merged**